java/mtsj/core/src/main/java/io/oasp/application/mtsj/bookingmanagement/logic/impl/BookingmanagementImpl.java

```diff
@@ -37,7 +37,7 @@
 import io.oasp.application.mtsj.bookingmanagement.logic.api.to.TableSearchCriteriaTo;
 import io.oasp.application.mtsj.general.common.api.constants.Roles;
 import io.oasp.application.mtsj.general.logic.base.AbstractComponentFacade;
-import io.oasp.application.mtsj.mailservice.api.Mail;
+import io.oasp.application.mtsj.mailservice.Mail;
```

Comment:
Not related to the actual refactoring but this is an invalid OASP package. We need to bring sonarqube-devon-plugin in place to avoid such things. Shouldn't a Mail component be a Devon/OASP standard module or what is individual here for MTS?